specs/core/0_beacon-chain.md

```diff
@@ -400,6 +404,19 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
+#### `WithdrawalData`
+
+```python
+{
+    # Withdrawal credentials
+    'withdrawal_credentials': 'hash32',
+    # Slot created
+    'slot_created': 'uint64',
+    # ETH to withdraw
+    'value': 'uint64'
```

Comment:
Rename `value` => `balance`

Reply:
If we do that, we should do the same for DepositData. Should we?

Reply:
We have three data structures for deposits: `Deposit`, `DepositData` and `DepositInput`. Definitely an opportunity for simplification/cleanup :)

Reply:
Maybe leave that for a separate patch at some point down the line? :smile: